client/src/components/QuestionsAndAnswers/index.jsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import styles from './QuestionsAndAnswers.css';
 import Question from '../Question';
 import Answer from '../Answer';
+import Helpful from '../Helpful';
 
 const QuestionsAndAnswers = () => (
   <div className={styles.qa}>
     <div className={styles.searchBar}>SearchBar</div>
+    <div className={styles.helpful}><Helpful /></div>
```

Comment:
<Helpful /> component returns its own containing div, so you could probably combine it and this div for less total divs. Just a suggestion for more elegant looking code but not a stopper by any means